app/views/coronavirus/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<% content_for :title, "Publish gov.uk/coronavirus content" %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <div class="govuk-body">
+      <p>Updating coronavirus landing page content:</p>
+      <ul>
```

Comment:
Shouldn't this be an ordered list as it's a step by step?

```suggestion
      <ol class="govuk-list govuk-list--number">
```

Reply:
👍